components/Checklist.js

```diff
@@ -0,0 +1,12 @@
+export default function Checklist({menuItems}, {menuName}) {
+    return (
+        <div>
+            <button type="button" class="checklist">{menuName}</button>
+            {menuItems.map((m) => (
+            <div>
+            <label>{m}</label>
+            <input type="checkbox" /><br /></div>
+     ))}
+     </div>
+    )
+}
```

Comment:
Use Alt + Shift + F to auto format code so this is easier to read. (You might need to double check the shortcut, not sure for Windows)